trainer.py

```diff
@@ -9,7 +9,6 @@
 - https://lightning.ai/docs/pytorch/2.1.0/cli/lightning_cli.html
 - https://pytorch-lightning.medium.com/introducing-lightningcli-v2-supercharge-your-training-c070d43c7dd6
 """
-import torch
```

Comment:
See example commit 012a7a8efda7b03dd3420bf3c8b10fe02d171317 where [`pre-commit-ci`](https://pre-commit.ci) automatically removed this unused import!